src/renderer/mirror-constants.ts

```diff
@@ -0,0 +1,16 @@
+import { Mirrors } from '../interfaces';
+
+export const ELECTRON_MIRRORS: {
+  DEFAULT: Mirrors;
+  CHINA: Mirrors;
+} = {
+  DEFAULT: {
+    electronMirror: 'https://github.com/electron/electron/releases/download/',
+    electronNightlyMirror:
+      'https://github.com/electron/nightlies/releases/download/',
+  },
+  CHINA: {
+    electronMirror: 'https://npmmirror.com/mirrors/electron/',
+    electronNightlyMirror: 'https://npmmirror.com/mirrors/electron-nightly/',
+  },
+};
```

Comment:
The code here cannot be placed in `constants.ts` because import `constants.ts` into the test will result in a large number of unit test errors.